src/mem.rs

```diff
@@ -136,7 +136,7 @@ impl MmapMemory {
 	/// memory, if the address is valid.
 	pub fn host_address(&self, addr: GuestPhysAddr) -> Result<*const u8, MemoryError> {
 		if addr < self.guest_address
```

Comment:
random sidenote for https://github.com/hermit-os/uhyve/issues/1081: this fails when `cmdval` is called